File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] ans = new int[2*n];
        int p = 0;
        for(int i = 0; i < n; i++) {
            ans[p] = nums[i];
            p++;
            ans[p] = nums[i+n];
            p++;
        }
        return ans;
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
    
    let i = 0, j = 0;
    let ans = new Array(2*n);
    while(i < n) {
        ans[j] = nums[i];
        ans[j+1] = nums[i+n];
        j += 2;
        i++;
    }
    return ans;
};